src/styles/utils/variables.scss

```diff
@@ -0,0 +1,4 @@
+$watch-size: 80vmin;
```

Comment:
The variables defined are correct as per the task description, but it's missing a variable for the second hand color.